core/src/mindustry/net/CrashSender.java

```diff
@@ -130,11 +144,11 @@ public static void send(Throwable exception, Cons<File> writeListener){
             boolean[] sent = {false};
 
             Log.info("Sending crash report.");
-            //post to crash report URL
+            //post to crash report URL, exit code indicates send success
             httpPost(Vars.crashReportURL, value.toJson(OutputType.json), r -> {
                 Log.info("Crash sent successfully.");
                 sent[0] = true;
-                System.exit(1);
+                System.exit(0);
```

Comment:
But the application still *crashed*, it should exit with 1 or at least a non-zero code.